tasks/coverage/parser_babel.snap

```diff
@@ -10004,16 +10005,6 @@ Expect to Parse: "typescript/types/const-type-parameters-babel-7/input.ts"
    ·       ╰── It can not be redeclared here
    ╰────
 
-  × Identifier `Context` has already been declared
```

Comment:
We got the wrong result if we skip the type only import

Reply:
It's fine for us to not report this typescript error, we are not a typescript checker after all ...

But it seems like we need to bind the symbols to the correct symbol flags at some point, but it's fine if this is skipped to get our transformer passed.

Reply:
By the way, symbol flag binding is directly copied from binder.ts, so you are free to modify the binder.